Automattic/sensei
# Add learner courses block

### Changes proposed in this Pull Request

* Includes the Learner Courses block.
* It introduces the `page` assets, for all blocks that can be used only in posts with `page` post type.
* It also introduces the `BundleAnalyzerPlugin`, so we can keep an eye on what's happening with the bundle sizes while working.

### Testing instructions

* Make sure you are able to add the Learner Courses block in any post.
* Make sure the other blocks continue restrict to their scope, like the Take Course block can only be added in the course page, and Contact Teacher block only in courses and lessons.
* Make sure that changing the settings and the layout view saves the attribute (after saving refresh the editor to confirm).

### Screenshots

<img width="1380" alt="Screen Shot 2021-03-24 at 17 41 08" src="https://user-images.githubusercontent.com/876340/112380454-29595400-8cc8-11eb-8bd5-73a319bf0b9f.png">
